src/Valet/Constants.cs

```diff
@@ -0,0 +1,42 @@
+namespace Valet;
+
+public static class Constants
+{
+    public static readonly List<string> UserInputVariables = new()
+    {
+        "GITHUB_ACCESS_TOKEN",
+        "GITHUB_INSTANCE_URL",
+        "AZURE_DEVOPS_ACCESS_TOKEN",
+        "AZURE_DEVOPS_PROJECT",
+        "AZURE_DEVOPS_ORGANIZATION",
+        "AZURE_DEVOPS_INSTANCE_URL",
+        "CIRCLE_CI_ACCESS_TOKEN",
+        "CIRCLE_CI_INSTANCE_URL",
+        "CIRCLE_CI_ORGANIZATION",
+        "CIRCLE_CI_PROVIDER",
+        "GITLAB_INSTANCE_URL",
+        "GITLAB_ACCESS_TOKEN",
+        "JENKINSFILE_ACCESS_TOKEN",
```

Comment:
Is there a way to attach help text to these lines? Some of these options are a bit vague such that I'm not sure a customer would know what some of them mean

Reply:
That's a good point. We could likely add some sort of "friendly" name/message to be used in the prompt that is more self describing

Reply:
Implemented in https://github.com/github/gh-valet/pull/45/commits/5ffbdc7a7fa728edb91484064e739fcb641480f4